code/game/verbs/ooc.dm

```diff
@@ -1,12 +1,16 @@
-/client/verb/ooc(message as text)
+/client/verb/ooc(message = "" as text)
 	set name = "OOC"
 	set category = "OOC"
 
-	sanitize_and_communicate(/decl/communication_channel/ooc, src, message)
+	if(!message)
+		message = input(src.mob, "", "ooc \"text\"") as text|null
```

Comment:
Maybe I'm misreading the diff but it seems this verb does nothing now.

Reply:
woops